benches/bench.rs

```diff
@@ -89,6 +89,78 @@ mod poly_benches {
     }
 }
 
+mod commitment_benches {
+    use super::*;
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
+
+    /// Benchmarks [de]ser of a univariate commitment
+    fn roundtrip(c: &mut Criterion) {
+        let mut rng = XorShiftRng::from_seed(RNG_SEED);
+        c.bench_function_over_inputs(
+            "Commitment [de]Serializiation",
+            move |b, &&deg| {
+                let rand_factors = || {
+                    let lhs = Poly::random(deg, &mut rng).commitment();
+                    let rhs = Poly::random(deg, &mut rng).commitment();
+                    let lhs_ser = bincode::serialize(&lhs).expect("lhs boom ser");
+                    let rhs_ser = bincode::serialize(&rhs).expect("rhs boom ser");
+                    (lhs_ser, rhs_ser)
+                };
+                b.iter_with_setup(rand_factors, |(lhs, rhs)| {
+                    (
+                        bincode::deserialize::<Commitment>(&lhs).expect("lhs boom deser"),
+                        bincode::deserialize::<Commitment>(&rhs).expect("rhs boom deser"),
```

Comment:
This bench is measuring deserialization only, but twice. Does that make sense in the context of this crate?

Reply:
nope, makes no sense. I'll remove